firmware/integration/rusefi_config.txt

```diff
@@ -932,7 +932,7 @@ bit useFSIO6ForRevLimiter;
 	adc_channel_e vRefAdcChannel;
 	uint8_t etbNeutralPosition;+Expected neutral position;"%",      1,      0,       0, 100,      0
 
-custom idle_mode_e 4 bits,    U32,   @OFFSET@, [0:0], "Automatic", "Manual"
+custom idle_mode_e 4 bits,    U32,   @OFFSET@, [0:0], "Closed Loop", "Open Loop"
```

Comment:
I am happy with 90% of this PR but not sure about this "Automatic" <> "Closed Loop"

"Closed Loop" is a smart terminology for engineers. If we can use terminology more understandable by more people I vote to go a bit less technical.